{{cookiecutter.project_slug}}/app.json

```diff
@@ -0,0 +1,49 @@
+{
+    "name": "{{ cookiecutter.project_name }}",
+    "env": {
+        "ALLOWED_HOSTS": {
+            "value": ".herokuapp.com"
+        },
+        "DEBUG": {
+            "value": "True"
+        },
+        "DISABLE_COLLECTSTATIC": {
+            "value": 1
+        },
+        "ENVIRONMENT": {
+            "value": "development"
+        },
+        "PROJECT_PATH": {
+            "value": "modat_api"
+        },
+        "NPM_CONFIG_PRODUCTION": {
+            "value": "false"
+        },
+        "SECRET_KEY": {
+            "generator": "secret"
+        }
+    },
+    "addons": [
+        "heroku-postgresql:standard-0",
+        "papertrail:Choklad"
+    ],
+    "environments": {
+        "review": {
+            "addons": [
+                "heroku-postgresql:hobby-dev",
+                "papertrail:Choklad"
```

Comment:
@oudeismetis what's the use case for papertrail in review apps?

Reply:
It's free and the best way to view logs from the app when you are debugging

Reply:
I haven't added it to my workflow, is it easier than `heroku logs -t`? That's what I usually use